lib/ansible/modules/database/mysql/mysql_variables.py

```diff
@@ -144,6 +178,7 @@ def main():
             ca_cert=dict(type='path', aliases=['ssl_ca']),
             connect_timeout=dict(type='int', default=30),
             config_file=dict(type='path', default='~/.my.cnf'),
+            mode=dict(type='str', choices=['global', 'persist', 'persist_only'], default='global'),
         ),
```

Comment:
Would be nice if the module could also support check mode (though that's for another PR). :)

Reply:
absolutely agree with you:) I noticed that it is not an attribute of all (or almost all) mysql modules unfortunately